contents/handbook/product/user-feedback.md

```diff
@@ -4,67 +4,52 @@ sidebar: Handbook
-- Not a hard rule, but it's usually good to limit the number of team members (around two) joining from our side to foment a more intimate chat. 
-
+- Add all feedback calls to the [User interviews calendar](https://calendar.google.com/calendar/?cid=Y19tczllaWN1Ym92ZGgxYWhzNmtoY2xpNTQ3b0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t). If the invite was created from your own calendar, you can simply add "User interviews" as an invitee.
 
 **During the call**
+- If you'd like to record the call, do ask before!
 - Do a quick round of intros if you haven't met previously.
-- Chat informally with the user to build rapport and gather some initial context (it will greatly improve the quality of the feedback!).
-- Ask the user if it's okay to record the conversation (mention we only use internally and keep for 3 months maximum).
 - If this is the first interview with the user, ask them for context about their company, their role, if they're technical.
-- The topics of the conversation are usually better informed from the previous step. It's always more helpful to talk about topics where the user has had more experience. It's however always useful to consider what we're currently working on at Product/Growth/Marketing and ask questions that could help each team do a better job.
-
 
 **After the call**
-1. Upload the recording to the [recordings folder][recordings], be sure to follow naming conventions.
-2. Add the notes to the [Google Doc][feedback-doc], linking the recording. Try to keep them as clean as possible.
-3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Orbit and/or Hubspot record in case Engineering needs more context when scoping/building.
-4. Generate the reward for the user (see below).
-   1. Most of the time, the reward will be a gift card for the [PostHog merch store](https://merch.posthog.com/). If it's the case, create the gift card in Shopify. Make sure to **add a link to their Orbit profile** in the notes section and set the **default expiration date** (5 years), and make note of the gift card code.
-5. Register call on Orbit. Remember to link the notes, add appropriate labels (e.g. `received merch`) and information on the rewards sent.
-6. If the customer is Sales/Scale, register the call on Hubspot too (but linking to Google doc notes).
-7. Follow-up with the user. Send any applicable rewards, links to any opened GitHub issues, and answers to any outstanding questions.
-
+1. Add the notes to the [Google Doc][feedback-doc], linking the recording if you did one.
+2. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.
```

Comment:
```suggestion
2. Share a short summary of the user interview in the `#posthog-feedback` Slack channel
3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.
```